test/performance/taeFakeAbiToFiscalCodeMap.js

```diff
@@ -0,0 +1,62 @@
+import { group } from 'k6'
+import exec from 'k6/execution'
+import { getAbiToFiscalCodesMap } from '../common/api/taeAbiToFiscalCodes.js'
+import { assert, bodyJsonSelectorValue, statusOk } from '../common/assertions.js'
+import {
+    isEnvValid,
+    isTestEnabledOnEnv,
+    DEV,
+    UAT,
+    PROD,
+} from '../common/envs.js'
+import dotenv from 'k6/x/dotenv'
+
+const REGISTERED_ENVS = [DEV, UAT, PROD]
+
+const services = JSON.parse(open('../../services/environments.json'))
+export let options = {
+    stages: [
+        { duration: '1m', target: 10 },
+        { duration: '3m', target: 30 },
+        { duration: '1m', target: 10 },
+    ],
+    thresholds: {
+        http_req_duration: ['p(95)<500'],
+    },
+}
+let params = {}
+let baseUrl
+let myEnv
+
+if (isEnvValid(__ENV.TARGET_ENV)) {
+    myEnv = dotenv.parse(open(`../../.env.${__ENV.TARGET_ENV}.local`))
+    baseUrl = services[`${__ENV.TARGET_ENV}_issuer`].baseUrl
+
+    options.tlsAuth = [
+        {
+            domains: [baseUrl],
+            cert: open(`../../certs/${myEnv.MAUTH_CERT_NAME}`),
+            key: open(`../../certs/${myEnv.MAUTH_PRIVATE_KEY_NAME}`),
+        },
+    ]
+
+    params.headers = {
+        'Ocp-Apim-Subscription-Key': myEnv.APIM_RTDPRODUCT_SK,
+    }
+}
+
+// In performance tests we shall use abort() to prevent the execution
+// of the default function, otherwise the VUs will be spawned
+if (!isTestEnabledOnEnv(__ENV.TARGET_ENV, REGISTERED_ENVS)) {
+    console.log('Test not enabled for target env')
+    exec.test.abort()
+}
+
+export default () => {
+    group('TAE AbiToFiscalCode API', () => {
+        group('Should retrieve conversion map', () =>
+            assert(getAbiToFiscalCodesMap(baseUrl, params), [statusOk(),
+            bodyJsonSelectorValue('STPAY', 'LU30726739')])
```

Comment:
@GiovanniMancini in your opinion does it make sense to have the second one assertion? That value right now is stubbed in dev/uat environment but it may change over time. Is it better to delete the assertion with this assumption?

Reply:
Anyway we may change the assertion at a later time